__mocks__/config.yml

```diff
@@ -9,7 +9,7 @@ ssl:
 protocols:
   http:
     enable: true
-    mocks_dir: "./"
+    mocks_dir: "./v1/"
```

Comment:
did you check this from postman? 
so in our front-end we use this link: `/api/v1/user/login`
in Postman you should be able to hit the same camouflage mock api with the request to `http://localhost:8080/v1/user/login` - almost the same but without `/api` prefix.

if that's not the case, 
please revert this line back to `mocks_dir: "./"`

Reply:
mocks_dir: "./v1/" - the request works as it should. Postman - http://localhost:8080/v1/user/login - do not work, 
http://localhost:3000/api/v1/user/login -  the request works as it should.

Reply:
please, try again, but do everything in excactly this order:
1. rebase your branch onto main
2. make sure you have this in vite.config.js
  `rewrite: (path) => path.replace(/^\/api\//, ''),`   —> No "v1" in this line
3. make sure, any api call you do in React/JS components has URI starting with: `/api/v1`
4. make sure, your `__mocks__/config.yml` has this: `mocks_dir: './'`

Reply:
Done